.github/actions/shared-env/action.yml

```
@@ -0,0 +1,204 @@
+      uses: actions/checkout@v3
+      with:
+        lfs: true
+
+    - name: Download workflow parameters
+      if: ${{ github.event.workflow_run }}
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: daily-build-dev
+        name: workflow-params
+        run_id: ${{ github.event.workflow_run.id }}
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)